backend/src/plugins/Automod/actions/changePerms.ts

```diff
@@ -0,0 +1,98 @@
+import { Permissions, PermissionString } from "discord.js";
+import * as t from "io-ts";
+import { automodAction } from "../helpers";
+import { tNullable, isValidSnowflake, tPartialDictionary } from "../../../utils";
+import { noop } from "knub/dist/utils";
+import { renderTemplate, TemplateSafeValueContainer } from "../../../templateFormatter";
+import {
+  guildToTemplateSafeGuild,
+  savedMessageToTemplateSafeSavedMessage,
+  userToTemplateSafeUser,
+} from "../../../utils/templateSafeObjects";
+
+export const ChangePermsAction = automodAction({
+  configType: t.type({
+    target: t.string,
+    channel: tNullable(t.string),
+    perms: tPartialDictionary(t.keyof(Permissions.FLAGS), tNullable(t.boolean)),
+  }),
+  defaultConfig: {
+    channel: "",
+    perms: {
+      SEND_MESSAGES: true,
+    },
+  },
```

Comment:
IMO this shouldn't have any defaults
```suggestion
  defaultConfig: {},
```